app/controllers/admin/routes.py

```diff
@@ -160,3 +158,27 @@ def addParticipants():
 
     return render_template('addParticipants.html',
                             title="Add Participants")
+
+@admin_bp.route('/courseManagement', methods = ['GET', 'POST'])
+@admin_bp.route('/<term>/courseManagement', methods = ['GET', 'POST'])
+def courseManagement(term = None):
+    '''
+    Renders the page for admins to manage Course Proposals
+    '''
+
+    term = Term.get_or_none(Term.id == term)
+    if not term:
+        term = g.current_term
+
+
+    pending, pendingCourseInstructor = pendingCourses(term)
+    approved, approvedCourseInstructor = approvedCourses(term)
```

Comment:
These return values should be combined. One is a list of course ids, and the other is a dictionary with keys that are course ids. That's the same data. If you return just the dictionary you have the data you need